src/UnityExtension/Assets/Editor/GitHub.Unity/UI/SettingsView.cs

```diff
@@ -94,6 +94,9 @@ public override void OnEnable()
         {
             base.OnEnable();
             AttachHandlers(Repository);
+
+            activeRemote = Repository != null ? Repository.CurrentRemote : null;
+            remoteHasChanged = true;
```

Comment:
Setting `remoteHasChanged` is not enough. We need to update `activeRemote` at some point.
Part of my thinking was that `activeRemote` was only read from here..

https://github.com/github-for-unity/Unity/blob/30e665cad007f96223c247237d9f2de830bb60cd/src/UnityExtension/Assets/Editor/GitHub.Unity/UI/SettingsView.cs#L229-L239